extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -208,6 +208,25 @@ export class ProjectsController {
 	public async buildProject(context: Project | dataworkspace.WorkspaceTreeItem): Promise<string> {
 		const project: Project = this.getProjectFromContext(context);
 
+		const telemetryProps: Record<string, string> = {};
+
+		let databaseSource: string | undefined;
+		let databaseSourceValues = project.getRawDatabaseSourceValues();
+
+		for (let databaseSourceIndex = 0; databaseSourceIndex < databaseSourceValues.length; databaseSourceIndex++) {
+			if (utils.IsWellKnownValue(databaseSourceValues[databaseSourceIndex])) {
+				if (!databaseSource) {
+					databaseSource = databaseSourceValues[databaseSourceIndex];
+				} else {
+					databaseSource += `;${databaseSourceValues[databaseSourceIndex]}`;
+				}
+			}
+		}
+
+		if (databaseSource) {
+			telemetryProps.databaseSource = databaseSource;
```

Comment:
It would be good to include this property regardless and just have it default to an empty string so that all instances of this event have the same schema